src/handwritten/task/_task.py

```diff
@@ -55,6 +55,10 @@ class Task:
     """
     Represents a DAQmx Task.
     """
+    __slots__ = ('_handle', '_close_on_exit', '_saved_name', '_grpc_options', '_event_handlers', '_interpreter',
```

Comment:
FYI, we should reformat this code with Black some time (not now) - https://github.com/ni/nidaqmx-python/issues/556